runtimes/standalone/src/lib.rs

```diff
@@ -341,15 +341,18 @@ impl pallet_sudo::Config for Runtime {
 }
 
 parameter_types! {
-	// TODO: Find reasonable number
 	pub const MaxDelegatedAttestations: u32 = 1000;
+	pub const AttestationDeposit: Balance = 100 * MILLI_KILT;
 }
 
 impl attestation::Config for Runtime {
 	type EnsureOrigin = did::EnsureDidOrigin<DidIdentifier, AccountId>;
 	type OriginSuccess = did::DidRawOrigin<DidIdentifier, AccountId>;
 	type Event = Event;
 	type WeightInfo = ();
+
```

Comment:
Q: Do we actually want a new line here?

Reply:
c6f6db615ff2c113755324d7a47012b250167d6c

I always like to separate types that are there for things that don't alter the logic of the pallet directly and stuff that changes how the pallet works